src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -75,6 +75,7 @@
     <PackageReference Include="Microsoft.Azure.AppService.Middleware.Functions" Version="1.5.4" />
     <PackageReference Include="Microsoft.Azure.AppService.Proxy.Client" Version="2.2.20220831.41" />
     <PackageReference Include="Microsoft.Azure.Cosmos.Table" Version="1.0.8" />
+    <PackageReference Include="Microsoft.Azure.Functions.Platform.Metrics.LinuxConsumption" Version="1.0.3" />
```

Comment:
For this draft PR, I manually uploaded this version of the package to the Azure Functions artifacts feed. An official build pipeline isn't set up for this yet.